### ISSUE TITLE
Keras Embedding layer docs missing "trainable" argument 

### ISSUE THREAD

**Issue by nsehwan** [keras-team/keras#15182]
**URL(s) with the issue:**.

https://keras.io/api/layers/core_layers/embedding/

**Description of issue (what needs to be changed):**.
Keras Embedding Layer API Docs https://keras.io/api/layers/core_layers/embedding/ do not mention the argument "trainable" in the Class definition. This argument can be useful for new users to know when deciding to train Keras embeddings, or use already pre-trained ones. Keras [blogs](https://blog.keras.io/using-pre-trained-word-embeddings-in-a-keras-model.html) and [project examples](https://keras.io/examples/nlp/pretrained_word_embeddings/) already refer to this argument in their code samples.

Please let me know if I should submit a PR for this instead, thanks !

**Parameters defined**.
Are all parameters defined and formatted correctly?
->> NO

**Usage example**
Is there a usage example?
->> YES (Already linked in description above).

**Submit a pull request?**.

Are you planning to also submit a pull request to fix the issue? See the [docs contributor guide](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md):
->> I can, will await response on issue.

**Comment by rthadur** [comment]
Hi @nsehwan yes you can add but I would suggest if you can add something similar to which is mentioned here about 'trainable' https://keras.io/api/layers/normalization_layers/batch_normalization/

cc @qlzh727

**Comment by qlzh727** [comment]
I think we didn't explicitly mention `training` in the call arg since it doesn't behave differently with the default layer behavior. In batch norm layer, since certain args will be updated differently (`mean` and `variance`), we explicitly call it out in the docstring.

For the default behavior of `trainiing`, you can find it in https://keras.io/api/layers/base_layer/.
